src/platform/web/Platform.js

```diff
@@ -130,9 +163,12 @@ export class Platform {
     }
 
     createAndMountRootView(vm) {
-        if (this.isIE11) {
+        if (this.isIE11 || this.isIOS) {
```

Comment:
I think we probably need a separate `ios` class for iOS rather than reusing `legacy`.

Reply:
Yes, you're right. :facepalm: 

Sorry, I'm going through the comments one by one and hadn't gotten to the one where the class is used yet.